internal/provider/log_stream_resource.go

```diff
@@ -183,6 +183,37 @@ func (r *logStreamResource) Schema(_ context.Context, _ resource.SchemaRequest,
 					},
 				},
 			},
+			"crowdstrike_http_event_collector": schema.SingleNestedAttribute{
+				Description: "Log Stream configuration for the Crowdstrike HTTP Event Collector (HEC) destination type.",
+				Optional:    true,
+				Attributes: map[string]schema.Attribute{
+					"hec_host_port": schema.StringAttribute{
+						Description: "Crowdstrike HTTP Event Collector (HEC) host:port value.",
+						Required:    true,
+						Validators: []validator.String{
+							validators.HecHostPortValidation(),
+						},
+					},
+					"api_key": schema.StringAttribute{
+						Description: "API Key.",
+						Sensitive:   true,
+						Required:    true,
+						Validators: []validator.String{
+							validators.ApiKeyValidation(),
+						},
+					},
+					"hec_source_name": schema.StringAttribute{
```

Comment:
```suggestion
					"source_name": schema.StringAttribute{
```
Since this is already in a block that references HEC, the prefix is unnecessary.